jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-backend/src/test/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImplTest.java

```diff
@@ -326,4 +331,78 @@ private CaseMilestone createTestMilestone(String caseMilestoneId, String caseMil
 
         return milestone;
     }
+
+    private TaskSummary createTaskSummary(Long taskId, String name, String status) {
+        TaskSummary taskSummary = TaskSummary.builder()
+                .id(taskId)
+                .name(name)
+                .status(status)
+                .createdOn(new Date())
+                .build();
+
+        return taskSummary;
+    }
+
+    private CaseAdHocFragment createAdHocFragment(String name, String type) {
+        CaseAdHocFragment caseAdHocFragment = CaseAdHocFragment.builder()
+                .name(name)
+                .type(type)
+                .build();
+
+        return caseAdHocFragment;
+    }
+
+    private void assertActionEquals(TaskSummary taskSummary, CaseActionSummary caseActionSummary) {
+        assertNotNull(taskSummary);
+        assertNotNull(caseActionSummary);
+        assertEquals(taskSummary.getId(), caseActionSummary.getId());
+        assertEquals(taskSummary.getName(), caseActionSummary.getName());
+        assertEquals(taskSummary.getStatus(), caseActionSummary.getStatus());
+    }
+
+    private void assertActionEqualsAdHocFragment(CaseAdHocFragment caseAdHocFragment, CaseActionSummary caseActionSummary) {
```

Comment:
This assertion method is practically the same as the one which was written in CaseActionAdHocMapperTest i.e. assertCaseActionAdHocFragment.
Can that method be somehow reused and avoid writing the new one with the same purpose?